flink-kubernetes-operator/src/main/resources/log4j2.properties

```diff
@@ -24,3 +24,6 @@ appender.console.name = ConsoleAppender
 appender.console.type = CONSOLE
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %style{%d}{yellow} %style{%-30c{1.}}{cyan} %highlight{[%-5level] [%X{resource.namespace}.%X{resource.name}] %msg%n%throwable}
+
+logger.conf.name = org.apache.flink.configuration.GlobalConfiguration
+logger.conf.level = WARN
```

Comment:
Why did you change this? Isn't it helpful to see what exact config is loaded by the operator?

Reply:
makes sense :)